website/docs/cloud-docs/integrations/service-now/service-graph/resource-coverage.mdx

```diff
@@ -0,0 +1,611 @@
+| Account Id        | Defaults to "VMware_vCenter"                                   |
+| Datacenter Type   | Defaults to "VMware_vCenter"                                   |
+| Object ID         | Defaults to "VMware_vCenter"                                   |
+| Name              | Defaults to "VMware_vCenter"                                   |
+| Operational Status| Defaults to "1" ("Operational")                                |
+
+#### VMware vCenter Datacenter (`cmdb_ci_vcenter_datacenter`)
+
+| CMDB field        | Terraform state field                                          |
+|-------------------|----------------------------------------------------------------|
+| Source Native Key | `datacenter_id`                                                |
+| Object Id         | `datacenter_id`                                                |
+| Region            | `datacenter_id`                                                |
+| Name              | `datacenter_id`                                                |
+| Operational Status| Defaults to "1" ("Operational")                                |
+
+#### VMware Virtual Machine Instance (`cmdb_ci_vmware_instance`)
+
+| CMDB field        | Terraform state field                                          |
+|-------------------|----------------------------------------------------------------|
+| Source Native Key | `id`                                                           |
+| Object Id         | `id`                                                           |
+| Name              | `name`                                                         |
+| Operational Status| Defaults to "1" ("Operational")                                |
+
+#### VMware vCenter Network (`cmdb_ci_vcenter_network`)
+
+| CMDB field        | Terraform state field                                          |
+|-------------------|----------------------------------------------------------------|
+| Source Native Key | `id`                                                           |
+| Object Id         | `id`                                                           |
+| Name              | `name`                                                         |
+| Operational Status| Defaults to "1" ("Operational")                                |
+
+#### VMware vCenter Datastore (`cmdb_ci_vcenter_datastore`)
+
+| CMDB field        | Terraform state field                                          |
+|-------------------|----------------------------------------------------------------|
+| Source Native Key | `id`                                                           |
+| Object Id         | `id`                                                           |
+| Name              | `name`                                                         |
+| Operational Status| Defaults to "1" ("Operational")                                |
+
+## Importing Tags
```

Comment:
Would we want to put this above the tables so it doesn't get lost in the scroll?

Reply:
Once I transferred the provider tables into their own separate child pages, the overview page got much nicer and shorter and no longer requires scrolling.